application/core/plugins/TwoFactorAdminLogin/helper/phpqrcode.php

```diff
@@ -157,7 +157,9 @@ public static function set(&$srctab, $x, $y, $repl, $replLen = false) {
  */
 
     class QRtools {
-    
+
+        public $frames;
```

Comment:
Need to separate this one ?

See https://bugs.limesurvey.org/view.php?id=19686

Reply:
Yep, should have been on that ticket. But let's leave it as it is